### DISCUSSION TITLE
Is filebrowser have activity log? (Such as IP logging, upload, download etc)

### DISCUSSION THREAD

**Discussion by c11161116** [filebrowser/filebrowser#1191] [Q&A]
Is filebrowser have activity log? (Such as IP logging, upload, download etc)
If yes, how can setup this? For current, I just see create account log.

Many thanks for you help.

**Comment by o1egl** [comment]
No it doesn't. As a workaround you can create a shell script and use it in `Settings -> Global Settings -> Command runner`
